Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/config/PortalSecurityConfig.java

```diff
@@ -61,10 +58,10 @@ public void configure(WebSecurity web) throws Exception {
         .antMatchers("/**/libs/**")
         .antMatchers("/**/js/*")
         .antMatchers("/portalPluginResource/**")
+        .antMatchers("/favicon.ico")
```

Comment:
This change would also have to be made in PortalKeyCloakSecurity.java, otherwise it'll only be effective when running without Keycloak.